src/libxtp/gwbse/rpa.cc

```diff
@@ -91,6 +91,46 @@ Eigen::MatrixXd RPA::calculate_epsilon(double frequency) const {
 template Eigen::MatrixXd RPA::calculate_epsilon<true>(double frequency) const;
 template Eigen::MatrixXd RPA::calculate_epsilon<false>(double frequency) const;
 
+Eigen::MatrixXcd RPA::calculate_epsilon_complex(double frequency_r,double frequency_i) const {
+  const Index size = _Mmn.auxsize();
+  std::vector<Eigen::MatrixXcd> thread_result = std::vector<Eigen::MatrixXcd>(
+      OPENMP::getMaxThreads(), Eigen::MatrixXcd::Zero(size, size));
```

Comment:
we use custom reductors now, please add one to `eigen.h`

Reply:
Tell me more

Reply:
If you look at the other RPA functions, they simply use a `reduce` keyword, for `MatrixXcd` this is not defined yet so you have to add that in the `eigen.h` header.

Reply:
Is this function needed at all any more?